### PR TITLE
Blog post with SWC/DC merger update

### PR DESCRIPTION
**Pull request by raynamharris**

### CODE REVIEW

File: _posts/2017-06-17-merger.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+categories:
+    - blog
+comments: true
+show_meta: true
+authors: ["Karen Cranston", "Rayna Harris", "Kate Hertweck", "Hilmar Lapp"]
+---
+
+On Jun 8 & 9, we met in-person to continue discussing how to proceed with the merger of the Software Carpentry and Data Carpentry organizations to best achieve our strategic goals and serve our communities. We are happy to report that is was a very productive meeting. 
+ 
+We spent most of the meeting discussing the structure of the future organizational leadership. We have drafted a number of motions that we will present to our respective boards for approval at our next board meetings. Among the motions, we will propose is that the combined Carpentries governing board structure be made up of both appointed and elected members. Additionally, we propose that it is in the best interest of the organization to seed the future governing board with bylaws, which will be enumerated, written, and approved in the coming months.
```
Comment (tracykteal) [inline _posts/2017-06-17-merger.md]: Steering Committees rather than boards.

Comment (tracykteal) [inline _posts/2017-06-17-merger.md]: "we will propose is" -> "we will propose that"

**Comment by tracykteal** [comment]
Thanks for posting this! We just can't call our governing bodies 'boards' since we're fiscally sponsored projects. For now they're Steering Committees. It could be called something else going forward, but still can't use the 'board' terminology. Can that change be made throughout?

**Comment by raynamharris** [comment]
Thanks @tracykteal. I've updated the terminology throughout.

**Pull request merged**